Felienne/hedy
# [FEATURE] Enables automatic generating of random passwords on multiple account creation

**Description**
In this PR we add a small feature to the multiple accounts creating process. Enabling teachers to generate random 6 char password for the accounts, making the process of creating multiple accounts easier. In combinatie with the downloading option of the account credentials this makes the process of adding multiple accounts really easy!

**Fixes**
This PR fixes #3186.

**How to test**
Make sure you are a teacher. Navigate to any class and select multiple account creation, verify that the random password generating works and valid passwords are being generated.
